src/select-field.jsx

```diff
@@ -51,6 +51,7 @@ const SelectField = React.createClass({
     menuItemStyle: React.PropTypes.object,
     selectedIndex: React.PropTypes.number,
     style: React.PropTypes.object,
+    labelMember: React.PropTypes.string,
```

Comment:
It's not implemented. Either implement this property and add it to the doc or let untouched this file.